common/startup/startup.go

```diff
@@ -170,6 +158,20 @@ func getNewJobStore() (job_store.JobStore, error) {
 	return store, nil
 }
 
+func handleSessionPushBack(session *session.Session) error {
```

Comment:
[nit] not a big fan of this helper's name, feels like we're using AWS terminology when we don't have to. What is this function actually doing in simpler terms?

Reply:
It's what actually enforces the time delay between each AWS request.

Reply:
then my minor suggestion would be something like `requestTimeDelay()` or `sessionTimeDelay()`

Reply:
:white_check_mark: That's reasonable.